next_frontend/.env.local

```diff
@@ -1,7 +1,7 @@
 # TODO(cgavidia): This should be handled via environment variables.
-# NEXT_PUBLIC_API_URL=http://localhost:8000
+NEXT_PUBLIC_API_URL=http://localhost:8000
```

Comment:
Please do not merge this code into `develop`

Reply:
@cptanalatriste good catch! Will switch this out, forgot that we had this in the repo.